src/dosext/mfs/lfn.c

```diff
@@ -1328,6 +1328,11 @@ int mfs_lfn(void)
 {
 	int carry, ret;
 
+	if (!mach_fs_enabled) {
+		CARRY;
+		return 0;
+	}
+
```

Comment:
I am not sure I understand this.
If DRDOS supplies NULL in cdsptr, then
how would this check save it from crashing
later?
Also I really think such check is too rude.
If lfn.c calls some functions from mfs.c, then
the check should probably be there.
Is it possible for lfn.c to not call directly
into mfs at all?